powerline/renderers/lemonbar.py

```diff
@@ -36,10 +37,26 @@ def hl(self, contents, fg=None, bg=None, attrs=None):
 		return text + contents + '%{F-B--u}'
 
 	def render(self, *args, **kwargs):
+		self.segment_info['output'] = kwargs.get('matcher_info')
```

Comment:
I'll do `kwargs['segment_info'] = {'output': kwargs.get('matcher_info')}` instead though, otherwise it will error if someone calls LemonbarRenderer.render(segment_info={})